src/main/java/menion/android/whereyougo/maps/utils/LocusMapDataProvider.java

```diff
@@ -75,13 +75,12 @@ public void addCartridges(Vector<CartridgeFile> cartridges) {
             }
 
             // construct waypoint
-            Location loc = new Location("WhereYouGo");
-            loc.setLatitude(cartridge.latitude);
-            loc.setLongitude(cartridge.longitude);
-            Waypoint wpt = new Waypoint(cartridge.name, loc);
-            wpt.addParameter(ExtraData.PAR_DESCRIPTION, cartridge.description);
-            wpt.addUrl(cartridge.url);
-            pack.addWaypoint(wpt);
+            Location loc = new Location(cartridge.latitude,cartridge.longitude);
+            Point wpt = new Point(cartridge.name, loc);
+            wpt.addParameter(GeoDataExtra.PAR_DESCRIPTION, cartridge.description);
+            wpt.addParameter(GeoDataExtra.PAR_SOURCE, cartridge.url);
+            //TODO wpt.addUrl(cartridge.url);
```

Comment:
The ``addUrl` was first migrated to `addParameterUrl`.
In version 0.9.50 this was moved to `GeoDataHelper`, 
see https://github.com/asamm/locus-api/commit/ecfd577b950780162d5bdd7f1505759618637645
So I asume that `GeoDataExtra.PAR_SOURCE` isn't correct.

Reply:
Used `GeoDataHelper` not working for me. In 0a12e7b151d35b4c236698e4288a36214d6d8c21 use `Point.setExtraData` directly.